server/base/src/main/java/org/apache/accumulo/server/util/Admin.java

```diff
@@ -96,26 +96,28 @@
 public class Admin implements KeywordExecutable {
   private static final Logger log = LoggerFactory.getLogger(Admin.class);
 
-  static class AdminOpts extends ServerUtilOpts {
-    @Parameter(names = {"-f", "--force"},
-        description = "force the given server to stop by removing its lock")
-    boolean force = false;
+  private static class SubCommandOpts {
```

Comment:
Do we need this class? Could the other commands extend `Help` like `ServerUtilOpts` does?

Reply:
> Do we need this class? Could the other commands extend `Help` like `ServerUtilOpts` does?

They could, but the subcommands aren't using the jcommander object that is in the Help command so it seemed like a waste.

Alternatively, we could also remove this class and add the help params to each of the sub commands directly.
It's just a couple more lines of code.

Reply:
I wouldn't put any more work into it, just wondering if we could re-use the `@Parameter` declaration.